applications/openshift/networking/default_ingress_ca_replaced/tests/ocp4/e2e-remediation.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-# You can safely patch this to a non-existent secret name, it's a no-op, but OK for testing.
-oc patch proxies.config cluster --type merge -p '{"spec":{"trustedCA":{"name":"testing"}}}'
+# You can safely patch this to a exist default secret name
```

Comment:
nit: maybe adding a quite note that a valid CA is required so this remediation doesn't cause subsequent failures.

Reply:
that's a good idea, just added some comments to it